packages/translation/src/base.ts

```diff
@@ -13,6 +13,8 @@ class NullTranslator implements ITranslator {
     this._languageBundle = bundle;
   }
 
+  readonly languageCode: string = 'en';
```

Comment:
Should we remove the existing `locale()` below to avoid confusion?

Reply:
Yes, that would be better.